src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/setup/ApiResponses.kt

```diff
@@ -192,6 +193,27 @@ fun assessmentsApiAssessmentsResponse(assessmentDate: LocalDateTime, assessmentI
 fun assessmentsApiNoSeverityNeedsResponse(): HttpResponse =
   assessmentApiResponse("no_severity_needs.json")
 
+fun tierToDeliusFullResponse(
+  gender: String = "Male",
+  currentTier: String = "UD0",
+  ogrsscore: String = 21.toString(),
+  rsrscore: String = 23.toString(),
+): HttpResponse =
+  jsonResponseOf(
+    responseFrom("$TIERTODELIUS_API_FIXTURES/tier-to-delius-response.json")
+      .replace("genderToReplace", gender)
+      .replace("tierToReplace", currentTier)
+      .replace("\"ogrsToReplace\"", ogrsscore)
+      .replace("\"rsrToReplace\"", rsrscore),
+  )
+
+fun tierToDeliusNoAssessResponse(gender: String, currentTier: String): HttpResponse =
```

Comment:
is there a reason this has no default values for parameters but the method above does?